sats-dna/src/main/kotlin/com/sats/dna/components/sessiondetails/SessionDetailsInfoLabel.kt

```diff
@@ -98,6 +109,70 @@ fun SessionDetailsInfoLabel(
     }
 }
 
+@Composable
+fun SessionDetailsSectionPlaceholder(
+    modifier: Modifier = Modifier,
+    showDurationLabel: Boolean = true,
+    showDateLabel: Boolean = true,
+    showLocationLabel: Boolean = true,
+    showWorkoutTypeLabel: Boolean = false,
+    showGxNameLabel: Boolean = false,
+) {
```

Comment:
Since the `SessionDetailsInfoSection` already just uses a bunch of slots to get its content, how about we instead make a `SessionDetailsInfoLabelPlaceholder` instead that we can pass into the slots when doing previews?